### ISSUE TITLE
Wrapper for script. Desktop application

### ISSUE THREAD

**Issue by TitaniumTosh** [k1tbyte/Wemod-Patcher#2]
How about creating some kind of interface to make the patcher easier to use?

**Comment by k1tbyte** [comment]
I'll think about the app if someone else needs it besides you. Anyway, you can always contribute

**Comment by Rogus9999** [comment]
> I'll think about the app if someone else needs it besides you. Anyway, you can always contribute

well I tried to git clone and it doesn't runs what do I need to execute it?

**Comment by Rogus9999** [comment]
please add a batch script or an executable. It's more user friendly.

**Comment by k1tbyte** [comment]
> > I'll think about the app if someone else needs it besides you. Anyway, you can always contribute
> 
> well I tried to git clone and it doesn't runs what do I need to execute it?

Make sure you have node js installed.

**Comment by Rogus9999** [comment]
> > > I'll think about the app if someone else needs it besides you. Anyway, you can always contribute
> > 
> > 
> > well I tried to git clone and it doesn't runs what do I need to execute it?
> 
> Make sure you have node js installed.

I just installed it and runs into the same problem do I need to use it via node.js .exe?
or via windows powershell?

**Comment by Rogus9999** [comment]
![image](https://github.com/user-attachments/assets/89b8d52f-fe20-4d4e-bb55-23bda192b162)
here's the error

**Comment by k1tbyte** [comment]
>  here's the error

Well, first you need to download the zip itself from GitHub (Code -> Download ZIP). 
1) After unzipping go to the Wemod-patcher directory (cd “path to folder”)
2) Install the dependencies using the npm install command
3) Run the script with the command: node ./unlocker.js.

In the screenshot you are not in the Wemod-patcher directory

**Comment by Rogus9999** [comment]
> > here's the error
> 
> Well, first you need to download the zip itself from GitHub (Code -> Download ZIP).
> 
>     1. After unzipping go to the Wemod-patcher directory (cd “path to folder”)
> 
>     2. Install the dependencies using the npm install command
> 
>     3. Run the script with the command: node ./unlocker.js.
> 
> 
> In the screenshot you are not in the Wemod-patcher directory

![image](https://github.com/user-attachments/assets/4da0dc69-53d3-4b85-bed8-6731eecfa1c5)
well I did as you suggested less the dependencies cause I don't know the npm install command.
it didn't even threw an error compreensive enough it's just about some asar cannot find module

**Comment by k1tbyte** [comment]
Cannot find the module because you have not installed the dependencies with the '**npm install**' command. npm is the default package manager. Make sure you have npm installed (default is selected with NodeJS installation)

![image](https://github.com/user-attachments/assets/c0dce90a-1360-4741-bebc-80d49ef74d98)

**Comment by Rogus9999** [comment]
![image](https://github.com/user-attachments/assets/c87d6590-1a67-4c83-be64-70dc1b54dbfb)
same error even after installing the node.js again without chocolatey
disregard it I got it working the command to install npm it's npm install without the quotes and you need to close the application to patch it wished that it could get it working the phone remote like one time ago the devs made a mistake or a way for reading whom's using the app cracked but it worked second the relates and reports of some people at cs.rin,ru .
Thanks for your patience do you make another applications or is it your first one?
I would like to support ya.
Do I have to always open terminal and execute in folder with files node ./unlocker.js ?

**Comment by k1tbyte** [comment]
I'm glad it worked out. Yes, I have other applications, you can see my projects in my profile. I wrote this script for myself and didn't think it would be useful for anyone. Maybe I will actually make an application to make the patch easier to use.

And yes, you should always call node ./unlocker.js but only if WeMod is updated.

If you want to support me you can use the button in the readme from the Support section.

**Comment by Rogus9999** [comment]
> I'm glad it worked out. Yes, I have other applications, you can see my projects in my profile. I wrote this script for myself and didn't think it would be useful for anyone. Maybe I will actually make an application to make the patch easier to use.
> 
> And yes, you should always call node ./unlocker.js but only if WeMod is updated.
> 
> If you want to support me you can use the button in the readme from the Support section.

it works but the application don't updates the cheats it gets on an always loading mods.
do I need to uninstall it and reinstall it?

**Comment by k1tbyte** [comment]
> > I'm glad it worked out. Yes, I have other applications, you can see my projects in my profile. I wrote this script for myself and didn't think it would be useful for anyone. Maybe I will actually make an application to make the patch easier to use.
> > And yes, you should always call node ./unlocker.js but only if WeMod is updated.
> > If you want to support me you can use the button in the readme from the Support section.
> 
> it works but the application don't updates the cheats it gets on an always loading mods. do I need to uninstall it and reinstall it?

You should just try to close WeMod and start again.  This can be avoided if you start the game directly from WeMod. I don't really know if it's related to the patch or a WeMod bug, but it happens sometimes

**Comment by Rogus9999** [comment]
ah one off topic question how much do you charge for an unpacker be made also repacker it's a tri-Ace PS2 Games UnPacker. if you are up to I can pay via ko-fi or via stripe pix since my currency is brl or brazilian real I cannot promise much and just if you want it too.
About the WeMod I just uninstalled and will reinstall. To see if the bug persists

**Comment by k1tbyte** [comment]
> ah one off topic question how much do you charge for an unpacker be made also repacker it's a tri-Ace PS2 Games UnPacker. if you are up to I can pay via ko-fi or via stripe pix since my currency is brl or brazilian real I cannot promise much and just if you want it too. About the WeMod I just uninstalled and will reinstall. To see if the bug persists

No, I can't help you with that question. I also don't ask for money for projects. All my projects are open source and available for free even if I ever decide to do something like what you mentioned ;)

**Comment by Rogus9999** [comment]
> > ah one off topic question how much do you charge for an unpacker be made also repacker it's a tri-Ace PS2 Games UnPacker. if you are up to I can pay via ko-fi or via stripe pix since my currency is brl or brazilian real I cannot promise much and just if you want it too. About the WeMod I just uninstalled and will reinstall. To see if the bug persists
> 
> No, I can't help you with that question. I also don't ask for money for projects. All my projects are open source and available for free even if I ever decide to do something like what you mentioned ;)

well the tool is already disponibilized only needs the reversing thing to make it repacker into the iso.
